### ISSUE TITLE
add gh_page for quality-requirements

### ISSUE THREAD

**Issue by gernotstarke** [arc42/quality-requirements#1]
arc42.github.io shall be the "landing page", linking to template, quality-requirements etc.

**Comment by gernotstarke** [comment]
or even better: http://quality-requirements.arc42.org

**Comment by aheusingfeld** [comment]
Is that still current? AFAICT arc42.github.io does not exist, so this probably needs a review

**Comment by gernotstarke** [comment]
you're right - arc42.github.io does not exist, and does not need to exist... the quality requirements should go elsewhere in the arc42 universe... and I will add a backlink from arc42.org once the html version has been published

**Comment by uniqueck** [comment]
So we would like push the html output as GitHub pages if a new release is triggered?

**Comment by gernotstarke** [comment]
sounds like a good idea... so we can link to this github-page from the arc42 template :-)

To make it perfect, we should create on overview page, not just the plain content...

Currently I've no good idea how to do separate structure from content: 
The current adoc files contain content-only - 
so a  new file/location for the overview (maybe containing the changelog) needs to be created.

I'm open for suggestions...

**Comment by gernotstarke** [comment]
see also #17...
